doc/src/tutorials/creating-packages.md

```diff
@@ -275,13 +275,17 @@ test-specific dependencies, are available, see below.
 
 ### Test-specific dependencies
 
-There are two ways of adding test-specific dependencies (dependencies that are not dependencies of the package but will still be available to
-load when the package is tested).
+There are two ways of adding test-specific dependencies (dependencies that are not dependencies of the package but
+will still be available to load when the package is tested). If no `test/Project.toml` exists Pkg will default to
+use the `target` based test specific dependencies.
 
 #### `target` based test specific dependencies
 
-Using this method of adding test-specific dependencies, the packages are added under an `[extras]` section and to a test target,
-e.g. to add `Markdown` and `Test` as test dependencies, add the following to the `Project.toml` file:
+!!! compat
```

Comment:
I would expect this `compat` notice *after* the explanation of what the `target` method does

Reply:
As this is directly related to the decision what functionality to choose, i could see that compat block being useful before further explanations.